tests/unit/pages/categories/Categories.spec.jsx

```diff
@@ -0,0 +1,21 @@
+import '@testing-library/jest-dom'
+import { screen } from '@testing-library/react'
+import Categories from '~/pages/categories/Categories'
+import { renderWithProviders } from '../../../test-utils'
+
+describe('Categories Component', () => {
+  it('renders "Categories" text within the PageWrapper', () => {
+    renderWithProviders(<Categories />)
```

Comment:
Maybe move it in `beforeEach`?

Reply:
![image](https://github.com/Space2Study-UA-1125/frontEnd/assets/93076661/e25f6ec1-11b3-4c20-afdb-c286c3a9ce25)
Should it look like this?